aldryn_newsblog/admin.py

```diff
@@ -45,6 +45,7 @@ class ArticleAdminForm(TranslatableModelForm):
 
     class Meta:
         model = models.Article
+        fields = '__all__'
```

Comment:
you can also specify the fields, but in order not to break anything I expose "__all__"